libraries/Microsoft.Bot.Builder.AI.QnA/Utils/QnACardBuilder.cs

```diff
@@ -77,48 +77,82 @@ public static IMessageActivity GetSuggestionsCard(List<string> suggestionsList,
         }
 
         /// <summary>
-        /// Get active learning suggestions card.
+        /// Get Card for MultiTurn scenario. (Can be deprected from 4.10.0 release of sdk).
         /// </summary>
         /// <param name="result">Result to be dispalyed as prompts.</param>
-        /// <param name="cardNoMatchText">No match text.</param>
+        /// <param name="cardNoMatchText">No match text.</param>       
         /// <returns>IMessageActivity.</returns>
         public static IMessageActivity GetQnAPromptsCard(QueryResult result, string cardNoMatchText)
```

Comment:
This doesn't make any sense. The method takes 2 parameters, but only uses one of them. 

Is this a bug, or an API design oversight?

Reply:
This is for resolving assembly error. Needs to be removed from next release.

Reply:
This is for resolving assembly error. Needs to be removed from next release.

---
In reply to: [430557744](https://github.com/microsoft/botbuilder-dotnet/pull/3935#discussion_r430557744) [](ancestors = 430557744)